packages/components/pie-checkbox/README.md

```diff
@@ -81,6 +81,8 @@ import { PieCheckbox } from '@justeattakeaway/pie-checkbox/dist/react';
 | `checked` | `boolean` | `false` | Indicates whether or not the checkbox is checked by default (when the page loads). |
 | `indeterminate` | `boolean` | `false` | Indicates whether the checkbox visually shows a horizontal line in the box instead of a check/tick. It has no impact on whether the checkbox's value is used in a form submission. That is decided by the checked state, regardless of the indeterminate state. |
 | `aria` | `object` | {} | accepts `label`, `labeledby` and `describedby` keys with string values. |
+| `assistiveText` | `string` | `''` | Allows assistive text to be displayed below the checkbox element. |
+| `status` | `'error'`, `'success'` | `undefined` | The status of the checkbox component / assistive text. If you use `status` you must provide an `assistiveText` prop value for accessibility purposes. |
```

Comment:
I think at the moment, the default value for `status` is `undefined`, which is used either when there is no assistive text or the message is neither a success nor an error.

We were seeing issues with `pie-text-input` in React apps when `status` only had two values: `"success"` and `"error"`. If you want to set the prop conditionally - which will happen very frequently - you have to specifically pass `undefined` in, which React doesn't like.

I've got a PR (#1513) open at the moment to see if adding a `"default"` status fixes it. If that works we might want to do the same thing for other components too.

Reply:
Xander described everything correctly. At the moment the `default` value for `status` is `undefined`, which is used either when there is **no assistive text or the message is neither a success nor an error.**
`pie-text-input` component works  with `status` in the same way.

Reply:
@leksaBoiko What would you think about trying to get #1513 merged, and then we could update this component to also have a `"default"` value for `status`, instead of `undefined`?

Reply:
This component was updated to also have a "default" value for status, instead of undefined

Reply:
Great stuff, thanks! Please can you update the README to reflect the updated prop values?

Reply:
great point, thank you!